### DISCUSSION TITLE
Is there a way to obtain the actual values versus the predicted values on the validation data?

### DISCUSSION THREAD

**Discussion by AhangarAamir** [autogluon/autogluon#4310] [Q&A]
Is there a way to obtain the actual values versus the predicted values on the validation data?

**Comment by shchur** [comment]
Hi @AhangarAamir, which module are you referring to? In case of time series, this discussion could be relevant https://github.com/autogluon/autogluon/issues/3525

**Reply by AhangarAamir** [reply]
@shchur I am using the TimeSeries module. Currently, I am hiding the latest data in the dataset, predicting on that, and then getting the predicted values (mean) as predicted and hidden data (target) as actual. However, if the data is daily for the last 10 years, I want the data on a yearly basis. AutoGluon automatically does the conversion, but when I hide the data, it remains as daily data. How can I convert this to yearly data? Please help.

**Reply by AhangarAamir** [reply]
@shchur I'm encountering a memory error with TemporalFusionTransformer. I have daily data consisting of 10,000 data points, with a prediction length set to 2,000 points, and I have data from the previous 10,000 days. Currently, I'm using 8 GB of memory for training the model, and my dataset size is in KBs with only two columns: target and date

**Reply by shchur** [reply]
Hi @AhangarAamir, could you provide a minimal reproducible example and, ideally, share the data with which you encountering this problem? It's hard to answer what exactly causes this issue otherwise.
